jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/LogGroupSelectorTable.kt

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+import com.intellij.ui.TableSpeedSearch
+import com.intellij.ui.TableUtil
+import com.intellij.ui.table.TableView
+import com.intellij.util.ui.ColumnInfo
+import com.intellij.util.ui.ListTableModel
+
+data class LogGroup(
+    var selected: Boolean = false,
+    var name: String? = null
+)
+
+class LogGroupSelectorTable : TableView<LogGroup>(model) {
+    init {
+        TableSpeedSearch(this)
+        TableUtil.setupCheckboxColumn(this, 0)
+        TableUtil.updateScroller(this)
```

Comment:
Does this table have a table header? If so, it can be rearranged and would be weird in this context